spec/config_spec.rb

```diff
@@ -0,0 +1,136 @@
+describe Logicware::Configurable do
```

Comment:
file name is wrong, but I would call the lib Logicware::Config
this is not a configurable, we are not adding this to models.

ideally we can get rid of the activesupport dep.